src/input/mpegts/iptv/iptv_rtsp.c

```diff
@@ -53,7 +53,7 @@ iptv_rtsp_alive_cb ( void *aux )
   iptv_mux_t *im = aux;
   rtsp_priv_t *rp = im->im_data;
 
-  rtsp_send(rp->hc, RTSP_CMD_OPTIONS, rp->path, rp->query, NULL);
+  rtsp_get_parameter(rp->hc, "");
```

Comment:
This is wrong for some RTSP servers. For example, SAT>IP specification defines only OPTIONS for the RTSP connection keep-alive signaling.